### DISCUSSION TITLE
Stream de alta qualidade da CNN Portugal não funciona

### DISCUSSION THREAD

**Discussion by JpbmOfficial** [LITUATUI/M3UPT#80] [General]
Olá.
Desde alguns dias a stream com maior qualidade da CNN Portugal não funciona. 
O programa inicia na stream com menor qualidade e quando tento mudar para a stream com maior, o programa não muda. Só quando mudo para alguma entre a segunda (com mais qualidade) e última (com menor qualidade) é que muda.

**Comment by LITUATUI** [comment]
Aqui na OTT Navigator funciona. Agora está a transmitir em HD e 1,8 Mbps.

No Yuki-IPTV também está na máxima resolução.

![yuki-iptv_CNN_Portugal](https://github.com/LITUATUI/M3UPT/assets/49299328/56f928b4-75e6-4ee3-9082-700ac6975309)

**Reply by cportugalr** [reply]
Eu estou a utilizar o iptv extreme, abre todos os canais Portugueses menos a Sic Notícias. Porque será?

**Reply by thomraider12** [reply]
> Eu estou a utilizar o iptv extreme, abre todos os canais Portugueses menos a Sic Notícias. Porque será?

Se abrires o link num browser, dá te erro. A muitos players também dá erro. Só alguns suportam.

**Reply by helio022** [reply]
acho que a sic noticias so funciona se tiveres em Portugal ou se usares uma vpn.

**Reply by alexptbg** [reply]
A sic noticias funciona aqui na Bulgaria, sem VPN.

**Reply by helio022** [reply]
entao nao sei porque nao funciona

**Comment by JpbmOfficial** [comment]
Estranho...
Eu tenho o Kodi na televisão e não funciona, e agora tentei no meu computador, no Kodi também, e acontece o mesmo.

**Reply by LITUATUI** [reply]
O link da CNN é HLS, ou seja, o bitrate adapta-se à velocidade da Internet. Pode ser que a ligação de Internet esteja com problemas.

Tenta o [speedtest](https://www.speedtest.net/pt) e tem atenção ao ping, quanto menor melhor.

**Reply by LITUATUI** [reply]
Testei agora no Kodi e fica na resolução mínima. 

O Kodi ultimamente está uma porcaria para IPTV...

**Reply by LITUATUI** [reply]
Alterei a lista para todos os canais excepto os canais da RTP, usarem o addon [InputStream Adaptive](https://github.com/xbmc/inputstream.adaptive).

Resolve o problema da resolução baixa e melhora a velocidade de zapping.

**Reply by JpbmOfficial** [reply]
É estranho como era só uma stream da CNN que era afetada... mas o que importa é que agora funciona! 
Antes todas as streams (pela box da minha televisão) costumavam ficar com um atraso no áudio e agora já não há esse atraso, não sei se é devido a essa mudança...
Mas obrigado!

**Comment by helio022** [comment]
uma pergunta para todos os que vivem no estrangeiro conseguem ver todos os canais portugueses?
Porque tenho o kodi na televisao e  a sic noticias nunca funciona desde que comecem a usar esta lista e a tvi e cnn apos 24 horas deixam de funcionar acontece isto a mais alguem?

**Reply by LITUATUI** [reply]
Para os canais geo-bloqueados como a SIC Notícias é preciso uma [VPN](https://m3upt.com/vpn).

https://github.com/LITUATUI/M3UPT#-notes

Os canais TVI e CNN Portugal têm links dinâmicos temporários, deve configurar a lista para ser actualizada de hora em hora.

**Reply by cportugalr** [reply]
Eu sei criar listas m3u8. Mas como você faz para os links atualizar sozinhos... pode me explicar se faz favor..

**Reply by helio022** [reply]
> Eu sei criar listas m3u8. Mas como você faz para os links atualizar sozinhos... pode me explicar se faz favor..

boa pergunta como actualizar os links sozinhos usando o kodi para google tv?

**Reply by LITUATUI** [reply]
O seu leitor de IPTV deverá ter uma opção para actualizar a lista de x em x horas.

**Comment by cportugalr** [comment]
Ok. A sua lista atualiza automaticamente. Eu queria criar a minha própria
lista e não dá. Eu se copiar os seus links e fazer uma lista nova pessoal
os links da RTP e TVI deixam de funcionar passado algum tempo.

A quinta, 1/08/2024, 14:27, Pedro Lima ***@***.***> escreveu:

**Reply by LITUATUI** [reply]
O [script](https://github.com/LITUATUI/M3UPT/blob/main/M3U/streams.sh) que actualiza os URLs dos streams automaticamente está no repositório.

**Reply by helio022** [reply]
obrigado senhor se nao for muito incomodo poderia dizer como utlizar esse script?
